learners/training_info.md

```diff
@@ -0,0 +1,101 @@
+A brief summary of these instructions is as follows
+
+1. Complete our Pre-training Survey. You will receive a custom link for your event when you receive your connection information.
+1. Select a lesson to use for teaching practice sessions and prepare a 3 minute segment, spending no more than 20-30 minutes to prepare.
+1. Please read the following:
+    * [The Science of Learning](https://carpentries.github.io/instructor-training/files/papers/science-of-learning-2015.pdf)
+    * [The Carpentries 2023 Annual Report](https://carpentries.org/files/reports/AnnualReport2023.pdf)
+
+### Checkout: The Instructor Certification Process
+After the training event, we ask you to complete three follow-up tasks to become a certified Instructor.
+These requirements are detailed on our [Checkout Instructions](../learners/checkout.md) page and will be discussed at our training.
+
+### What to Bring to an In-Person Event
+[Most Instructor Training events take place online](#where).
+If your training will take place in-person, please bring a laptop that is Internet connected and has a functioning browser. 
+If you have it, a device for recording audio and video (mobile phones and laptops are OK) is useful as we are going to record one another teaching in pairs or threes. 
+It does not have to be high-quality, but it should be good enough that you can understand what someone is saying.
+
+### Attendance and Cancellation
+Trainees who miss more than 1 hour of the training may be marked absent.
+Instructor certification cannot be completed without full attendance at an Instructor Training event. 
+If you unexpectedly need to miss more than 1 hour of your event, please contact your Trainers using the details provided in the information email for your event or [email us](mailto:instructor.training@carpentries.org).
+
+For events in which registration occurs through The Carpentries via Eventbrite, cancellation may be performed in Eventbrite up to the start of the event.
+Canceled seats cannot be filled after the 1 week registration deadline for these events, so we ask that you only cancel if absolutely necessary.
+
+More information on our [cancellation and makeup policy](https://docs.carpentries.org/topic_folders/instructor_training/cancellations_and_makeups.html) is available in The Carpentries Handbook.
+
+### Contact
+Please contact your Trainers using the details provided in the information email for your event or [email us](mailto:instructor.training@carpentries.org) with questions about Instructor Training.
+
+
+## Training Materials and Schedule
+This website contains the curriculum for Instructor Training.
+Use the left sidebar to browse the content, and visit [the landing page in Instructor View](https://carpentries.github.io/instructor-training/instructor/index.html#schedule) for a sample schedule for a 2-day event.
+Note that many Instructor Training events take place over four half-days, and this schedule will not apply in those cases.
```

Comment:
the schedule actually just times things out over ~16 straight hours , but all 4 parts are indicated. I have opened a [PR proposing adding explanatory text](https://github.com/carpentries/instructor-training/pull/1733/files) on the linked page, but we should rephrase here too (at a minimum to reflect that the schedule is over 16 hours not 2 days)

Reply:
```suggestion
Use the left sidebar to browse the content, and visit [the landing page in Instructor View](https://carpentries.github.io/instructor-training/instructor/index.html#schedule) for a sample schedule.
Most Instructor Training events take place over four half-days, and the majority of others span two full days.
```

Good point, thanks @brownsarahm. How's this?

Reply:
this looks good to me